Helper.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+public class Helper
+{
+    protected List<char> _listOfLettersToCount;
```

Comment:
not sure what are you referring to  ? do i need to make it private or protected ?

Reply:
I see that you're using protected with one list and private with the other. Perhaps it would be better if you keep it the same, because it seems that the lists will be used in the same way. Which to choose depends on the accessibility level you want to have:

- Private - The type or member can be accessed only by code in the same class or struct.
- Protected - The type or member can be accessed only by code in the same class, or in a class that is derived from that class.

(https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers))